search/tests/tests.py

```diff
@@ -0,0 +1,1442 @@
+            },
+            "name"
+        )
+        self.assertEqual(field_value, "Come and listen to my story")
+
+        field_value = _find_field(
+            {
+                "name": {
+                    "first": "Martyn",
+                    "last": "James"
+                }
+            },
+            "name.first"
+        )
+        self.assertEqual(field_value, "Martyn")
+
+        field_value = _find_field(
+            {
+                "name": {
+                    "first": "Monica",
+                    "last": {
+                        "one": "Parker",
+                        "two": "James"
+                    }
+                }
+            },
+            "name.last.two"
+        )
+        self.assertEqual(field_value, "James")
+
+        with self.assertRaises(ValueError):
+            field_value = _find_field(
+                {
+                    "name": "Come and listen to my story"
+                },
+                123
+            )
+
+        with self.assertRaises(ValueError):
+            field_value = _find_field(123, "name")
+
+    def test_filter_optimization(self):
+        """ Make sure that intersection optimizes return when no filter dictionary is provided """
+        test_docs = [{"A": {"X": 1, "Y": 2, "Z": 3}}, {"B": {"X": 9, "Y": 8, "Z": 7}}]
+        self.assertTrue(_filter_intersection(test_docs, None), test_docs)
+
+    def test_datetime_conversion(self):
+        """ tests json_date_to_datetime with different formats """
+        json_date = "2015-01-31"
+        self.assertTrue(json_date_to_datetime(json_date), datetime(2015, 1, 31))
+
+        json_datetime = "2015-01-31T07:30:28"
+        self.assertTrue(json_date_to_datetime(json_datetime), datetime(2015, 1, 31, 7, 30, 28))
+
+        json_datetime = "2015-01-31T07:30:28.65785"
+        self.assertTrue(json_date_to_datetime(json_datetime), datetime(2015, 1, 31, 7, 30, 28, 65785))
+
+        json_datetime = "2015-01-31T07:30:28Z"
+        self.assertTrue(json_date_to_datetime(json_datetime), datetime(2015, 1, 31, 7, 30, 28))
+
+        json_datetime = "2015-01-31T07:30:28.65785Z"
+        self.assertTrue(json_date_to_datetime(json_datetime), datetime(2015, 1, 31, 7, 30, 28, 65785))
+
+
+@override_settings(SEARCH_ENGINE="search.tests.tests.ForceRefreshElasticSearchEngine")
+class ElasticSearchTests(MockSearchTests):
+    """ Override that runs the same tests for ElasticSearchEngine instead of MockSearchEngine """
+    pass
+
+
+@override_settings(MOCK_SEARCH_BACKING_FILE="./testfile.pkl")
+class FileBackedMockSearchTests(MockSearchTests):
+    """ Override that runs the same tests with file-backed MockSearchEngine """
+
+    def setUp(self):
+        MockSearchEngine.create_test_file()
+        self._searcher = None
+
+    def tearDown(self):
+        MockSearchEngine.destroy_test_file()
+        self._searcher = None
+
+    # Commenting test for now - it runs locally, but not on travis system
```

Comment:
Is there a decorator we can use here to skip the test and have to have to comment it out. I believe there is a @skipTest or something like that we've used elsewhere.